mail_footer_notified_partners/models/mail_followers.py

```diff
@@ -48,16 +48,17 @@ def get_signature_footer(
 
         return footer
 
-    @api.model
+    @api.multi
```

Comment:
this function cannot be `multi` because `message_id` is not an id of a notification, but of a message